### PR TITLE
Applicants project grants mobile sections

### PR DESCRIPTION
**Pull request by nhsz**
This PR adds the main sections for `/applicants/project-grants` (mobile version)

PREVIEW: https://deploy-preview-116--thirsty-meitner-cae269.netlify.app/applicants/project-grants

- `PageSection` component added to `UI/headings`
- page sections content added (except Process and FAQ, which will be added on separated PR's)
- minor refactor on some components props
- fonts files added
- `global.css` styles added to override Chakra's theme heading font-family for certain headings

### CODE REVIEW

File: src/components/UI/headings/PageSection.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+export const PageSection: FC<HeadingProps> = ({ children, ...props }) => {
+  return (
+    <Heading
+      as='h3'
+      color='#e44550'
+      fontSize='22px'
+      fontWeight={700}
+      lineHeight='29px'
+      className='maison-neue-mono'
```
Comment (pettinarip) [inline src/components/UI/headings/PageSection.tsx:12]: Just an idea. Perhaps we could define this in the theme to keep everything scoped there and avoid creating custom css classes.

We could do something like:
```ts
extendTheme({
  ...
  components: {
    Heading: {
      variants: {
        'page-section': {
          fontFamily: 'Maison Neue Mono'
        }
      }
    }
  }
});

// and then in here, just
<Heading variant="page-section" ...>
```

**Review by pettinarip — APPROVED** [review]
LGTM!

File: src/components/UI/headings/PageSection.tsx (hunk 1 above)
Comment (nhsz) [reply src/components/UI/headings/PageSection.tsx]: Fixed! I wasn't able to make it work [using a Font component](https://chakra-ui.com/guides/recipes/using-fonts#option-2-using-font-face) like the Chakra guide suggests, but the variant works with the font-face declared on the `global.css` file.

Comment (pettinarip) [reply src/components/UI/headings/PageSection.tsx]: Great! gj

**Review by pettinarip — APPROVED** [review]

**Pull request merged**